.kitchen.ec2.yml

```diff
@@ -16,7 +16,7 @@ transport:
 
 provisioner:
   name: chef_zero
-  require_chef_omnibus: latest
```

Comment:
@jeremyfryer, you made this change because the default for this provisioner setting is true?